Source/ThirdParty/ANGLE/.gitmodules

```diff
@@ -1,11 +1,47 @@
+[submodule "build"]
+	path = build
+	url = https://chromium.googlesource.com/chromium/src/build.git
+	gclient-condition = not build_with_chromium
+[submodule "buildtools"]
+	path = buildtools
+	url = https://chromium.googlesource.com/chromium/src/buildtools.git
+	gclient-condition = not build_with_chromium
 [submodule "third_party/clang-format/script"]
 	path = third_party/clang-format/script
 	url = https://chromium.googlesource.com/external/github.com/llvm/llvm-project/clang/tools/clang-format.git
 	gclient-condition = not build_with_chromium
+[submodule "testing"]
+	path = testing
+	url = https://chromium.googlesource.com/chromium/src/testing
+	gclient-condition = not build_with_chromium
+[submodule "third_party/abseil-cpp"]
+	path = third_party/abseil-cpp
+	url = https://chromium.googlesource.com/chromium/src/third_party/abseil-cpp
+	gclient-condition = not build_with_chromium
+[submodule "third_party/android_build_tools"]
+	path = third_party/android_build_tools
+	url = https://chromium.googlesource.com/chromium/src/third_party/android_build_tools
+	gclient-condition = checkout_android and not build_with_chromium
+[submodule "third_party/android_deps"]
+	path = third_party/android_deps
+	url = https://chromium.googlesource.com/chromium/src/third_party/android_deps
+	gclient-condition = checkout_android and not build_with_chromium
+[submodule "third_party/android_platform"]
+	path = third_party/android_platform
+	url = https://chromium.googlesource.com/chromium/src/third_party/android_platform
+	gclient-condition = checkout_android and not build_with_chromium
+[submodule "third_party/android_sdk"]
+	path = third_party/android_sdk
+	url = https://chromium.googlesource.com/chromium/src/third_party/android_sdk
+	gclient-condition = checkout_android and not build_with_chromium
```

Comment:
Did you trim these out before? I reverted the additions in my change to remove them. Maybe that's what caused the issues with my previous roll.

Reply:
I haven't treated the file any different to other files.

This time, I imported the upstream version of the file for this update, in order to remove all our "changes" to it.